private/dependency_tree_parser.bzl

```diff
@@ -78,6 +80,13 @@ def _generate_imports(repository_ctx, dep_tree, explicit_artifacts, neverlink_ar
                     if repository_ctx.attr.maven_install_json:
                         all_imports.append(_genrule_copy_artifact_from_http_file(artifact))
 
+    jetify_all = repository_ctx.attr.jetify and repository_ctx.attr.jetify_include_list == JETIFY_INCLUDE_LIST_JETIFY_ALL
+
+    # Write artifacts to dict to achieve O(1) lookup instead of O(n).
+    jetify_include_dict = {}
+    for jetify_include_artifact in repository_ctx.attr.jetify_include_list:
+        jetify_include_dict[jetify_include_artifact] = None
```

Comment:
Welp, not sure that creating a dict is cheaper than searching in a list but I’ll bite.